src/components/context/CardContext.js

```diff
@@ -0,0 +1,40 @@
+import { createContext, useState } from 'react';
+import initialCards from '../../cards-data';
+const GameContext = createContext();
+
+const GameProvider = ({ children }) => {
```

Comment:
This `context` folder should be inside of your `src` folder, not inside of `components`.